meteor/client/ui/Settings/ShowStyleBaseSettings.tsx

```diff
@@ -1466,7 +1466,7 @@ const ShowStyleVariantsSettings = withTranslation()(
 			const { t } = this.props
 			return (
 				<div>
-					<h2 className="mhn">{t('Variants')}</h2>
+					<h2 className="mhn">{t('Showstyle Variants')}</h2>
```

Comment:
English user-facing strings generally spell it as _Show Style_. I think we should be consistent, one way or another. (But that's more of a comment to the creator of the task)

Reply:
Yeah, i did notice that it was written in multiple ways throughout, i talked to Rasmus and ultimately went with the manner it was written in the task. Should i change it to "Show Style" instead?